src/terrain/routes.clj

```diff
@@ -64,6 +64,11 @@
         (tc/with-logging-context {:user-info (cheshire/encode user-info)}
           (handler request))))))
 
+(defn optionally-authenticated-routes
+  []
+  (util/flagged-routes
+   (secured-filesystem-routes)))
```

Comment:
For the time being, filesystem routes are going to be available both  inside and outside the `/secured` context. When these routes are called outside the `/secured` context, authentication will only be required for routes that need it. When they're called inside the `/secured` context, authentication will always be required. This allows us to keep supporting the existing DE without encountering the paradox of calling a `/secured/...` endpoint with no authentication information. 😆